nut/rootfs/etc/cont-init.d/nut.sh

```diff
@@ -11,6 +11,7 @@ declare shutdowncmd
 declare upsmonpwd
 declare username
 
+mkdir -p /var{/run/nut,/state/ups}
```

Comment:
You should use `/run/nut` rather than `/var/run/nut` (since `/var/run` is a symlink to `/run`).

What is /var/state/ups used for?

Reply:
That's correct, I just followed whatever was in the script. I've updated the original code as well.

Reply:
`/var/state/ups` is for the active driver state. In my case, the folder contains:
```
upssched - empty dir
usbhid-ups-bx950mi - fifo
usbhid-ups-bx950mi.pid
```

Reply:
Ok.  `/var/state/ups` doesn't seem to be used for my use case, but fine if it is used for other use cases.

Should you set permissions on it as well?

Reply:
Yes, but why was it needed in the first place? You can't chown to root without root permissions, and there is no reason to limit the access permissions since those files are not secret.